lib/utils/__tests__/parseSelectorAST.test.mjs

```diff
@@ -0,0 +1,48 @@
+import postcss from 'postcss';
+
+import parseSelectorAST from '../parseSelectorAST.mjs';
+
+describe('parseSelectorAST', () => {
+	it('parses selectors into an AST', async () => {
+		const result = await postcss().process('a .foo, b::before {}');
+		const rule = result.root.first;
+		const ast = parseSelectorAST(rule.selector, result, rule);
+
+		expect(ast).toBeTruthy();
+		expect(result.warnings()).toHaveLength(0);
+	});
+
+	it('short circuits on empty selectors', async () => {
```

Comment:
nit

This describes the internals.
i.e. I would expect a `toThrow`

Reply:
I am unsure if your comment is about the message for this test, the test itself, or the implementation of `lib/utils/parseSelectorAST.mjs`.

Reply:
It's about the string passed as the first argument to the `it` function.

The _name_ should match the test and not the implementation.
i.e. in the case of a refactor—where the return type doesn't change—the test name shan't be changed
It's non-blocking, you can disregard it if you are disagreeing.

Reply:
I've changed the first description to :

> returns `undefined` when there is no selector